src/App.js

```diff
@@ -1,13 +1,29 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 import SearchbarContainer from "./components/SearchBar/SearchbarContainer";
+import PostContainer from "./components/Post Container/PostContainer";
+import dummyData   from "./dummy-data";
+import CommentSectionContainer from "./components/CommentSection/CommentSectionContainer";
 
 class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      data:[dummyData]
```

Comment:
Wrapping dummy data in an array may be the reason for your problems. You can input dummydata without the brackets.